x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -142,6 +143,41 @@ const assertResponse = (resp: CspSetupStatus, logger: CspApiRequestHandlerContex
   }
 };
 
+const checkIndexHasFindings = async (
+  esClient: ElasticsearchClient,
+  index: string,
+  logger: Logger
+) => {
+  try {
+    const response = await esClient.search({
+      index,
+      size: 0, // We only need to know if there are any hits, so we don't need to retrieve documents
+      query: {
+        bool: {
+          filter: [
+            {
+              range: {
+                '@timestamp': {
+                  gte: `now-${LATEST_FINDINGS_RETENTION_POLICY}`,
```

Comment:
should we also include the retention policy as a function parameter? so the function will be truly generic to other use cases such as vulnerabilities.